projects/valtimo/process-link/src/lib/components/select-form/select-form.component.ts

```diff
@@ -134,8 +136,10 @@ export class SelectFormComponent implements OnInit, OnDestroy {
           id: selectedProcessLink.id,
           formDefinitionId: this.selectedFormDefinition.id,
           viewModelEnabled,
-          formDisplayType: this.formDisplayValue,
-          formSize: this.formSizeValue,
+          ...(this.configService.featureToggles?.enableTaskPanel && {
```

Comment:
Put this.configService.featureToggles?.enableTaskPanel in a private attribute and use it instead of calling for the service everytime